src/analysis/retail/evoker/preservation/normalizers/CastLinkNormalizer.ts

```diff
@@ -660,6 +802,24 @@ const EVENT_LINKS: EventLink[] = [
       return c.hasTalent(TALENTS_EVOKER.ANCIENT_FLAME_TALENT);
     },
   },
+  {
+    linkRelation: T31_2PC,
+    reverseLinkRelation: T31_2PC,
+    linkingEventId: [
+      SPELLS.FIRE_BREATH.id,
+      SPELLS.FIRE_BREATH_FONT.id,
+      TALENTS_EVOKER.DREAM_BREATH_TALENT.id,
+      TALENTS_EVOKER.SPIRITBLOOM_TALENT.id,
+      SPELLS.DREAM_BREATH_FONT.id,
+      SPELLS.SPIRITBLOOM_FONT.id,
+    ],
+    linkingEventType: [EventType.EmpowerEnd, EventType.Cast],
+    referencedEventId: [SPELLS.LIVING_FLAME_DAMAGE.id, SPELLS.LIVING_FLAME_HEAL.id],
+    referencedEventType: [EventType.Damage, EventType.Heal],
+    maximumLinks: T31_LF_AMOUNT,
+    forwardBufferMs: EB_BUFFER_MS * 2, // travel time489
+    anyTarget: true,
```

Comment:
Need to filter out leaping flames procs here, since they would both proc from Fire Breath.
Maybe a combo of
`!HasRelatedEvent(referencedEvent, LEAPING_FLAMES) && HasRelatedEvent(referencedEvent, ECHO_TIER_HEAL)` ?